src/admin/python/lsst/qserv/admin/cli/entrypoint.py

```diff
@@ -409,7 +433,7 @@ def proxy(
     show_default=True,
 )
 @targs_options()
-@cmd_option(default="cmsd -c {{cmsd_manager_cfg_path}} -n manager -I v4")
+@cmd_option(default=commands["cmsd-manager"].default_cmd)
```

Comment:
As far as I can see, the only for the `cmd_option` decorator is to inject the default command template. All uses are in the current nodule. Why don't do this here and in the rest of the module?
```
@cmd_option("cmsd-manager")
```
The implementation of the decorator would pull the collection of the commands and find the right default for the specified command....in case if your implementation permits it.

Reply:
`cmd_option` is simply a `click.option` with some vars predefined/bound (much like C++ `std::bind`).
But it's an interesting idea to move the use of `commands` into the implementation. I _can_ inherit from `click.Option`, I'll see if your idea can be implemented in a simple way.